### PR TITLE
Add cloudfront distribution to app

### PR DESCRIPTION
**Pull request by mmmmillar**
WHAT: Add a cloudfront distribution to our app via Serverless

- Also bumped the authorizer version as the other version was broken

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -84,11 +84,59 @@ resources:
...
+      Properties:
+        DistributionConfig:
+          Aliases:
+            - ${self:custom.aliases.${self:provider.stage}}
+          PriceClass: PriceClass_100
+          ViewerCertificate:
+            AcmCertificateArn: arn:aws:acm:us-east-1:402949050862:certificate/1ab6042c-b6df-40f9-abdc-9ab8d8df1679
+            MinimumProtocolVersion: TLSv1.2_2018
+            SslSupportMethod: sni-only
+          DefaultCacheBehavior:
```
Comment (bjpirt) [inline serverless.yml:98]: Does this disable all caching? Wondering if it would be possible to only diable on /index.html ?

File: serverless.yml [hunk 2]
```diff
@@ -84,11 +84,59 @@ resources:
...
 
+    CloudFrontDistribution:
+      Type: AWS::CloudFront::Distribution
+      Properties:
+        DistributionConfig:
+          Aliases:
+            - ${self:custom.aliases.${self:provider.stage}}
+          PriceClass: PriceClass_100
+          ViewerCertificate:
+            AcmCertificateArn: arn:aws:acm:us-east-1:402949050862:certificate/1ab6042c-b6df-40f9-abdc-9ab8d8df1679
```
Comment (t-davies) [inline serverless.yml:95]: Are we cool with making the AWS account id public?

File: serverless.yml (hunk 1 above)
Comment (t-davies) [reply serverless.yml:98]: This app isn't a SPA and so the caching rules would need to be more complex than that. I agree with not disabling all caching at the CloudFront level though, I _think_ we should be fine to leave out `MaxTTL` and `MinTTL` then at least if we do set caching headers on static content or other things in our API they'll be respected.

File: serverless.yml [hunk 3]
```diff
@@ -84,11 +84,59 @@ resources:
...
+              - OPTIONS
+              - PUT
+              - PATCH
+              - POST
+              - DELETE
+            DefaultTTL: 0
+            MaxTTL: 0
+            MinTTL: 0
+            ForwardedValues:
+              QueryString: false
```
Comment (t-davies) [inline serverless.yml:113]: Are we sure we'll not need the querystring forwarded?

File: serverless.yml (hunk 1 above)
Comment (mmmmillar) [reply serverless.yml:98]: Yep, will address the caching stuff in a future PR when we have an app running :)

File: serverless.yml (hunk 3 above)
Comment (mmmmillar) [reply serverless.yml:113]: Not sure no

File: serverless.yml (hunk 1 above)
Comment (bjpirt) [reply serverless.yml:98]: 👍

**Pull request merged**